projects/ngx-quill/package.json

```diff
@@ -31,9 +31,9 @@
   ],
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=13.0.0",
+    "@angular/core": ">=14.0.0",
     "quill": "^1.3.7",
-    "rxjs": "^6.5.3 || ^7.4.0"
+    "rxjs": ">=7.0.0"
```

Comment:
could you please use ^

i do not know possible breaking changes with angular 15 and above. And there are many new deprecation with rxjs 7 and angular 14 is limited to rxjs 7 i guess